lib/ui/widgets/dialogs/theme_dialog.dart

```diff
@@ -70,16 +67,8 @@ extension ThemePickerItemExt on PickerItem {
     ThemeOptions themeOption,
   ) {
     final themeSetting = ThemeSetting(themeOption);
-    if (themeOption == ThemeOptions.flat ||
-        themeOption == ThemeOptions.byzantineVioletFlat ||
-        themeOption == ThemeOptions.darkFlat ||
-        themeOption == ThemeOptions.emeraldGreenFlat ||
-        themeOption == ThemeOptions.honeyOrangeFlat ||
-        themeOption == ThemeOptions.navyBlueFlat ||
-        themeOption == ThemeOptions.pearlGreyFlat ||
-        themeOption == ThemeOptions.fireRedFlat ||
-        themeOption == ThemeOptions.sapphireBlueFlat ||
-        themeOption == ThemeOptions.seaGreenFlat) {
+    // Flat theme
+    if (themeOption.toString().toLowerCase().endsWith('flat')) {
```

Comment:
Simplifying this code is a pretty good idea 👍

I'm not an absolute fan of relying on the `ThemeOptions` values name. Indeed, those values are not strictly formatted.
What about adding a `ThemeOptions::isFlat` property (using [Dart enhanced enums](https://dart.dev/language/enums#declaring-enhanced-enums)) ?

@gdaguin This is a discussion-opened suggestion.